float64[] joint_configs_on_way 
bool ifSave
---
float64[] pos
float64[] vel
float64[] acc
float64[] t